migration/migration/008-install-uda.sql

```diff
@@ -56,10 +56,22 @@ $func$
 LANGUAGE sql STABLE;
 GRANT EXECUTE ON FUNCTION _prom_catalog.is_multinode() TO prom_reader;
 
+CREATE OR REPLACE FUNCTION _prom_catalog.is_restore_in_progress()
```

Comment:
If you create this here, then you will also need to create it in the 0.0.0 script. We need to be careful about the changes that we make the existing `migration/migration` scripts.

Reply:
good catch

Reply:
removed it from here and left it only in idempotent script